cluster/schema/snapshot.go

```diff
@@ -65,20 +70,7 @@ func (s *schema) Persist(sink raft.SnapshotSink) (err error) {
 	return nil
 }
 
-// Release is invoked when we are finished with the snapshot.
-func (s *schema) Release() {}
-
-func snapshotIndex(ss *raft.FileSnapshotStore) uint64 {
```

Comment:
**question (non-blocking):** intended delete ?

Reply:
Moved to the snapshot subpart of schema package.